src/tseda/datastore.py

```diff
@@ -184,23 +220,38 @@ def __panel__(self):
             formatters=self.formatters,
             editors=self.editors,
             margin=10,
-            text_align={"selected": "center"},
+            text_align={col: "left" for col in self.columns},
+            header_filters=filters,
         )
         return pn.Column(self.tooltip, table)
 
-    def sidebar(self):
+    def options_sidebar(self):
         return pn.Card(
```

Comment:
is it possible to zoom out on the map? If I select eg only sample 1, I get I zoomed in version of the map focusing on these samples. and I can't figure out how to zoom out

Reply:
Also: the map doesn't update automatically when I change the selected sample sets, only the table

Reply:
You have to click the scrolling tool to zoom out or in, and I have made a new branch to work on the map table connection it is called individuals_map_table_connection :D